Typo fix in txzchk

xexyl opened the pull request:
Changed MIN_TXZCHK_VERSION to "2.1.0 2025-11-18".
Updated TXZCHK_VERSION to "2.1.1 2025-12-20"`.

IMPORTANT NOTES: this is NOT a functional change! It is NOT necessary to have this before IOCCC30 and this is one of the reasons I changed the minimum version to what is now the previous version of txzchk. This is only done because it annoys me - NOT because it's important to anyone but me (and it's certainly unimportant for the contest). In other words even if this is not merged (or if it is not installed on the submit server) until after IOCCC29 it does not matter in the slightest. There is no real need other than to make me happier about the typo being killed.

... in fact although it has no functional change and although the min version should be fine I would personally not install this on the server until the next time - but that's me. I leave that to Landon nonetheless.

xexyl committed:
Typo fix in txzchk

Changed MIN_TXZCHK_VERSION to "2.1.0 2025-11-18".
Updated TXZCHK_VERSION to "2.1.1 2025-12-20"`.

IMPORTANT NOTES: this is NOT a functional change! It is NOT necessary to
have this before IOCCC30 and this is one of the reasons I changed the
minimum version to what is now the previous version of txzchk. This is
only done because it annoys me - NOT because it's important to anyone
but me (and it's certainly unimportant for the contest). In other words
even if this is not merged (or if it is not installed on the submit
server) until after IOCCC29 it does not matter in the slightest. There
is no real need other than to make me happier about the typo being
killed.

... in fact although it has no functional change and although the min
version should be fine I would personally not install this on the server
until the next time - but that's me. I leave that to Landon nonetheless.

lcn2 reviewed (approved):
👍

lcn2 reviewed (changes requested):
There are merge conflicts with this PR, and as such this PR needs to be modified.

xexyl committed:
Merge branch 'master' into typo

xexyl committed:
Update CHANGES.md for merge conflict

This will be a series of commits on files (CHANGES.md and soup/version.h) to try and fix merge conflicts (as there are multiple pull requests).

xexyl committed:
Update version.h

Update repo version to try and get merge conflicts resolved in a pull request. More to follow (today or another day).

xexyl committed:
Update CHANGES.md

It appears that the previous commit on this file was in error (but working this out). It seems more likely that this file was not  updated for the commit in question (a typo fix in txzchk). The changes added in the previous commit were for a later commit.

lcn2 reviewed (approved):
thank you @xexyl 👍

This PR will be processed after the winning entries of **IOCCC29** have been published.
